Makefile

```diff
@@ -592,21 +593,27 @@ scale-image: scale-build clean-image
 	cp bin/linux/profiler scale/image/bin/profiler
 	cp bin/linux/chaos scale/image/bin/chaos
 	chmod +w scale/image/bin/*
-	docker build -t stackrox/scale:$(TAG) -f scale/image/Dockerfile scale
-	docker tag stackrox/scale:$(TAG) quay.io/$(CI_QUAY_REPO)/scale:$(TAG)
+	docker build \
+		-t stackrox/scale:$(TAG) \
+		-t quay.io/rhacs-eng/scale:$(TAG) \
+		-f scale/image/Dockerfile scale
 
 webhookserver-image: webhookserver-build
 	-mkdir webhookserver/bin
 	cp bin/linux/webhookserver webhookserver/bin/webhookserver
 	chmod +w webhookserver/bin/webhookserver
-	docker build -t stackrox/webhookserver:1.2 -f webhookserver/Dockerfile webhookserver
-	docker tag stackrox/webhookserver:1.2 quay.io/$(CI_QUAY_REPO)/webhookserver:1.2
+	docker build \
+		-t stackrox/webhookserver:1.2 \
+		-t quay.io/rhacs-eng/webhookserver:1.2 \
+		-f webhookserver/Dockerfile webhookserver
```

Comment:
Why `-t quay.io/rhacs-eng/webhookserver:1.2` instead of `$(DEFAULT_IMAGE_REGISTRY)/webhookserver:1.2`? Are the webhookserver images always put in rhacs-eng even for open source builds?

Reply:
1. Former `docker build` and `docker tag` commands will be equivalent to extra `-t ...` argument to `docker build`.
2. If you check what was `CI_QUAY_REPO` before, you'd find that it was either empty or `rhacs-eng`. Empty would result in incorrect repositories `quay.io//webhookserver` which means to me that these images were never pushed like that, only with CI_QUAY_REPO set to `rhacs-eng`.